tests/v2/test_waku_rln_relay.nim

```diff
@@ -579,8 +579,14 @@ suite "Waku rln relay":
     # verify the proof
     let verified = rln.proofVerify(data = messageBytes,
                                     proof = proof)
+
+    # Ensure the proof verification did not error out
+
+    check:
+      verified.isOk()
+
     check:
-      verified == true
+      verified.value() == true
```

Comment:
You can group all the checks together:
```
check:
      verified.isOk()
      verified.value() == true
```

Reply:
Addressed in [fefb7e7](https://github.com/status-im/nwaku/pull/1158/commits/fefb7e706182ee964835d1393c5a9f7dc480ad9d)